README.md

```diff
@@ -3,7 +3,7 @@ The official mobile app of the University of Illinois. Powered by the [Rokwire P
 
 ## Requirements
 
-### [Flutter](https://flutter.dev/docs/get-started/install) v3.3.2
+### [Flutter](https://flutter.dev/docs/get-started/install) v3.10.2
```

Comment:
Please sync the branch with the latest from `rokwire/illinois-app` `develop` branch. For example, current flutter version that we use for building the app is `3.13.7`

Reply:
done